src/graph.rs

```diff
@@ -1594,6 +1596,133 @@ impl PyGraph {
         Ok(out_dict.into())
     }
 
+    /// Docs Pending
+    ///
+    #[pyo3(
+        text_signature = "(self, node, other, edge_map_fn, /, node_filter=None, edge_weight_map=None"
+    )]
+    fn substitute_node_with_subgraph(
+        &mut self,
+        py: Python,
+        node: usize,
+        other: &PyGraph,
+        edge_map_fn: PyObject,
+        node_filter: Option<PyObject>,
+        edge_weight_map: Option<PyObject>,
+    ) -> PyResult<NodeMap> {
+        let filter_fn = |obj: &PyObject, filter_fn: &Option<PyObject>| -> PyResult<bool> {
+            match filter_fn {
+                Some(filter) => {
+                    let res = filter.call1(py, (obj,)).unwrap();
```

Comment:
Any reason for using `unwrap` here instead of `?` like in the digraph version?

Reply:
No particular reason. Rust was just complaining about using them without an extra import. I guess it was a local problem because it works well now. I will revert back to using the `?`. Thank you for reviewing.